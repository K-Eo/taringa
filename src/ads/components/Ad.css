.Ad-empty {
  min-height: 270px;
}
