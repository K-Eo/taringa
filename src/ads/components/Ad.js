import React from 'react'

import './Ad.css'

const Ad = props => {
  return (
    <div className="card">
      <div className="card-header p-1 small text-center text-muted">
        PUBLICIDAD
      </div>
      <div className="card-body Ad-empty" />
    </div>
  )
}

export default Ad
