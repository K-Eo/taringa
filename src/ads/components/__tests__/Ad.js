import React from 'react'
import Ad from '../Ad'

describe('Ad', () => {
  it('exists', () => {
    expect(Ad).toBeDefined()
  })

  it('renders', () => {
    const wrapper = shallow(<Ad />)

    expect(wrapper).toMatchSnapshot()
  })
})
