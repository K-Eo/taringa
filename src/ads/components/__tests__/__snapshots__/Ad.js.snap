// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ad renders 1`] = `
<div
  className="card"
>
  <div
    className="card-header p-1 small text-center text-muted"
  >
    PUBLICIDAD
  </div>
  <div
    className="card-body Ad-empty"
  />
</div>
`;
